recipe/meta.yaml

```diff
@@ -1,17 +1,16 @@
 {% set name = "kedro" %}
-{% set version = "0.18.3" %}
-
+{% set version = "0.18.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/quantumblacklabs/kedro/archive/{{ version }}.tar.gz
-  sha256: 596959f80ebbeef06234eeb0f2f8916ed7fdc8c0af5a5aae55835a2c88dd914c
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kedro-{{ version }}.tar.gz
+  sha256: 8556a7b5d9cdbea674c152a55779a471ff53064650eaca9660ff8276c96bed68
```

Comment:
Just out of curiosity: how does this `sha` get generated? It's different from the one in the PR opened automatically: https://github.com/conda-forge/kedro-feedstock/pull/26/files

Reply:
Its ok to use the github tarball, generally the recommendation on conda-forge is to use the PyPi tarball - So I tried changing it, We can change it back to the github tarball if you'd prefer that. 
The sequence of recommended sources I believe is PyPi tar, git hub tar, and then github repo. 
If all else fails, github repo + patches in the recipe directory. 
For SHA - https://pypi.org/project/kedro/#files
If click on view hashes - you can see the sha

Reply:
Thanks for explaining! I'm more than happy to change it to the PyPi tarball.